cmd/snap/cmd_snap_op.go

```diff
@@ -94,6 +94,11 @@ have developer access to the snap, either directly or through the
 store's collaboration feature, and to be logged in (see 'snap help login').
 
 Note a later refresh will typically undo a revision override.
+
+Holds (--hold) specified for snaps are only applied to general refreshes ('snap 
+refresh' without specifying snaps and auto-refreshes), but snaps can still be
+refreshed in specific refreshes (i.e, 'snap refresh snap-a snap-b'). Using
+--hold without specifying any snaps will only hold auto-refreshes for all snaps.
```

Comment:
I've tried tweaking this slightly to hopefully make our intentions clearer (I left the time specifics and syntax to the command options list):
```suggestion
Hold (--hold) is used to postpone general snap refresh updates for either all
snaps, or for the specified snaps. It is only effective with auto-refreshes and
refreshes that do not target snaps specifically. If a snap is first held but
then added to a refresh command, such as `snap refresh target-snap`, it will
still be refreshed.
```

Reply:
@degville I think in the reformulation is lost that "snap refresh --hold" blocks only auto-refreshes and not "snap refresh" (no specific snap) itself.

Reply:
Thanks @pedronis, I'd forgotten that it didn't block a general snap refresh. Does this make it clearer?

```suggestion
Hold (--hold) is used to postpone automatic snap refresh updates for all snaps
when no snaps are specified, or for the specified snaps. It is only effective
with auto-refreshes and will not block either general refresh requests from
`snap refresh` or specific snap requests from `snap refresh target-snap`.
```

Reply:
It does block general refreshes for specific snaps. So I still don't think I read that in the new version. Maybe the easiest is really to have two paragraphs, one about "snap refresh --hold"  and a separate one about "snap refresh --hold <snap>"?

Reply:
I tried to split into a general/intro paragraph and then one paragraph about when no snaps are specified and one for snaps being specified.